laplace/baselaplace.py

```diff
@@ -679,6 +680,15 @@ def _nn_predictive_samples(self, X, n_samples=100, **model_kwargs):
             fs = torch.softmax(fs, dim=-1)
         return fs
 
+    def _nn_bma_clf(self, X, n_samples=100, **model_kwargs):
```

Comment:
I would prefer a more explicit name for this function, e.g. `_nn_predictive_classification`.